{
	"id": "CVE-2023-49080",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2023-49080",
				"title": "Jupyter Server errors include tracebacks with path information",
				"description": "The Jupyter Server provides the backend (i.e. the core services, APIs, and REST endpoints) for Jupyter web applications like Jupyter notebook, JupyterLab, and Voila. Unhandled errors in API requests coming from an authenticated user include traceback information, which can include path information. There is no known mechanism by which to trigger these errors without authentication, so the paths revealed are not considered particularly sensitive, given that the requesting user has arbitrary execution permissions already in the same environment. A fix has been introduced in commit `0056c3aa52` which no longer includes traceback information in JSON error responses. For compatibility, the traceback field is present, but always empty. This commit has been included in version 2.11.2. Users are advised to upgrade. There are no known workarounds for this vulnerability.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "GitHub_M",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:L/I:N/A:N",
							"base_score": 3.5,
							"base_severity": "LOW",
							"temporal_score": 3.5,
							"temporal_severity": "LOW",
							"environmental_score": 3.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "GitHub_M",
						"cwe": [
							"CWE-209"
						]
					}
				],
				"references": [
					{
						"source": "CVE",
						"url": "https://github.com/jupyter-server/jupyter_server/commit/0056c3aa52cbb28b263a7a609ae5f17618b36652"
					},
					{
						"source": "CVE",
						"url": "https://github.com/jupyter-server/jupyter_server/security/advisories/GHSA-h56g-gq9v-vc8r"
					},
					{
						"source": "CVE",
						"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/62LO7PPIAMLIDEKUOORXLHKLGA6QPL77/"
					},
					{
						"source": "CVE",
						"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FG2JWZI5KPUYMDPS53AIFTZJWZD3IT6I/"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/jupyter-server/jupyter_server/commit/0056c3aa52cbb28b263a7a609ae5f17618b36652"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/jupyter-server/jupyter_server/security/advisories/GHSA-h56g-gq9v-vc8r"
					},
					{
						"source": "GitHub_M",
						"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/62LO7PPIAMLIDEKUOORXLHKLGA6QPL77/"
					},
					{
						"source": "GitHub_M",
						"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FG2JWZI5KPUYMDPS53AIFTZJWZD3IT6I/"
					}
				],
				"published": "2023-12-04T21:00:59.026Z",
				"modified": "2024-08-02T21:46:28.982Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2023/CVE-2023-49080.json"
		]
	}
}
